maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -161,7 +167,9 @@ class TestCommand : Callable<Int> {
                     debugOutputPath = debugOutputPath
                 )
 
-                TestDebugReporter.deleteOldFiles()
+                if(!flattenDebugOutput){
```

Comment:
nit: the code format seems wrong, there is no space between `if` and the `(` and between `)` and the `{`.